### ISSUE TITLE
ChatCo no longer working properly with NoEncryption

### ISSUE THREAD

**Issue by lerokko** [Doclic/NoEncryption#98]
### Are you using the latest version of NoEncryption?

- [X] I am using the latest version of NoEncryption compatible with my server version.

### Is there an existing issue for this?

- [X] I have searched the existing issues and have not found a duplicate relating to my issue.

### NoEncryption Version

5.0

### Minecraft Version

1.19.4

### Server Type/Build

This server is running Paper version git-Paper-509 (MC: 1.19.4) (Implementing API version 1.19.4-R0.1-SNAPSHOT) (Git: e338793)

### Plugins in Use

ChatCo, ChatEx, TAB, Luckperms

### Reproducible Test

I am not sure if this is the right place to ask but players are getting kicked with
`Received chat packet with missing or invalid signature.`
This happens when they send a message with ChatCo (using /msg) and then sent a public chat message afterwards.

### Anything else?

I have secure profiles, secure chat, and whatever I can turn off turned off. For all I care the signature could be the entire bee movie script dear minecraft server just let the man send the message and do not kick him. (I am not blaming NoEncryption)
I don't know why it is happening but you are my only hope since ChatCo is no longer maintained (for 6 years). It used to worked fine on all the versions of NoEncryption prior it seemed that recently (In the last months or so) it just started happening to some players.

**Comment by doclic** [comment]
I have tried to check ChatCo's source code, but it is very hard to read, and frankly terrible.
As you said yourself, ChatCo is old and hasn't been updated in 6 years, so you should probably use modern plugins such as [ChatCoPlus](https://github.com/zeroBzeroT/ChatCoPlus) instead.

**Comment by V1nc3ntWasTaken** [comment]
I am going to have to go with the above comment. ChatCo is very outdated and uses APIs from 1.11.

**Comment by lerokko** [comment]
I tried using ChatCoPlus but that just overrides any chat plugin and makes it the vanilla chat...
I have now resorted to splitting the task to two plugins BukkitGreentext and MSG to get mostly feature parity to what I required from ChatCo originally. It is not as elegant but it seems to work fine and I will push it to production soon.

ChatCoPlus not respecting other chat formats has nothing to do with NoEncryption so feel free to close this issue.
